LensmaniaServer/Docs/PostsApi.md

```
@@ -0,0 +1,92 @@
+| Parameter | Type  | Required | Default | Description                                           |
+|-----------|-------|----------|---------|-------------------------------------------------------|
+| `limit`   | `int` | No       | `10`    | Number of posts to return per page                    |
+| `cursor`  | `int` | No       | `null`  | Id of the last post received, used to load more posts |
+
+---
+
+## Examples
+
+### First page (without cursor)
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)